org.recoll.recoll.yml

```diff
@@ -0,0 +1,125 @@
+id: org.recoll.recoll
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: io.qt.qtwebkit.BaseApp
+base-version: 5.15-23.08
+command: recoll
+rename-desktop-file: recoll-searchgui.desktop
+rename-icon: recoll
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=home
+  - --env=ASPELL_CONF=dict-dir /app/share/dicts; local-data-dir /app/share/datadicts
+
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+
+modules:
+  - name: xapian-core
+    config-opts:
+      - --disable-sse
+      - --disable-backend-chert
+      - --disable-backend-remote
+      - --disable-documentation
+    cleanup:
+      - /bin
+      - /share/aclocal
+      - /share/doc
+      - /share/man
+    sources:
+      - type: archive
+        url: https://oligarchy.co.uk/xapian/1.4.26/xapian-core-1.4.26.tar.xz
+        sha256: 9e6a7903806966d16ce220b49377c9c8fad667c8f0ffcb23a3442946269363a7
+
+  - name: chmlib
+    cleanup:
+      - /include
+      - '*.a'
+    sources:
+      - type: archive
+        url: http://www.jedrea.com/chmlib/chmlib-0.40.tar.bz2
+        sha256: 3449d64b0cf71578b2c7e3ddc048d4af3661f44a83941ea074a7813f3a59ffa3
+      - type: script
+        commands:
+          - autoreconf --install
+
+  - name: dictEn-aspell
+    buildsystem: simple
```

Comment:
this should use autotools, `make-install-args` exists.

Reply:
they are probably using a non standard configure script that doesn't accept prefix.

Reply:
Regardless of how we provide these options, the core issue is that it does not accept the `--prefix` argument. Therefore, I believe the most straightforward solution for now is just use `buildsystem: simple` demonstrated in [org.texmacs.TeXmacs.yml#L49](https://github.com/flathub/org.texmacs.TeXmacs/blob/7aec1bd74e6d1459a02693154435ec98c37fe41f/org.texmacs.TeXmacs.yml#L49).